lpa-submissions-e2e-tests/cypress/integration/enterAppealSite/enterAppealSite.step.js

```diff
@@ -2,7 +2,7 @@ import { Before } from 'cypress-cucumber-preprocessor/steps';
 
 const page = {
   id: 'enterAppealSite',
-  heading: 'Would the inspector need to enter the appeal site?',
+  heading: 'Would the Inspector need to enter the appeal site?',
   section: 'About the Appeal Site',
   title:
     'Would the Inspector need to enter the appeal site? - Appeal Questionnaire - Appeal a householder planning decision - GOV.UK',
```

Comment:
Check with @pippalou if we need to change 'inspector' to 'Inspector' in the error message? on Line 10 of this file. I can't add a comment there

Reply:
@shilpabasu22 Yes please,  good spot

Reply:
I have updated the content